### PR TITLE
[v7r0] document DErrno codes; some cleanup of CommandReference files, add anchors -- Merge order: 3

### PR DESCRIPTION
**Pull request by andresailer**
I now removed all command rst files that are fully re-created via the script, the remaining ones contain examples _only_ in the rst file. (This will be yet another large PR)

As requested by @chaen  it is now possible to add some code to dynamically create documentation for the Code parts. E.g. DErrno:

https://diracsailer.readthedocs.io/en/fixdoc/CodeDocumentation/Core/Utilities/DErrno.html 
https://diracsailer.readthedocs.io/en/fixdoc/_sources/CodeDocumentation/Core/Utilities/DErrno.rst.txt

- [x] The docs fail because of #3979 

BEGINRELEASENOTES

*Docs:
NEW: Add reference anchor to command descriptions, commands in the AdministratorGuide are pre-fixed with 'admin_' because there would be duplicate anchors otherwise. 
NEW: Add infrastructure to manually append documentation to the CodeDocumentation
CHANGE: When running MakeDoc or buildScriptsDocs directly they only replace files with changed content, greatly speeding up the incremental builds
CHANGE:  MakeDoc or buildScriptsDocs now accept -dd and -ddd flags to switch to more verbose output via the command line

ENDRELEASENOTES

### CODE REVIEW

File: docs/Tools/buildScriptsDocs.py [hunk 1]
```diff
@@ -14,6 +14,8 @@
 import subprocess
 import shlex
 
+SUPER_DEBUG = False
```
Comment (chaen) [inline docs/Tools/buildScriptsDocs.py]: Maybe just a comment to explain this global variable ?

Comment (andresailer) [reply docs/Tools/buildScriptsDocs.py]: Done

**Comment by andresailer** [comment]
Rebased, also fixed the new documentation warnings

**Comment by atsareg** [comment]
After merging the #3975 this one has conflicts unfortunately. @andresailer : can you please rebase again and fix the conflicts

**Comment by andresailer** [comment]
Ok, fixed conflicts.

**Comment by andresailer** [comment]
I removed the ExampleConfig.cfg , because that is created fully automatically

**Pull request merged**